src/commands/paint.ts

```diff
@@ -97,15 +97,19 @@ export function paint(
     process.stdout.write(`${chalk.bold('Snowpack')}\n\n`);
     // Dashboard
     if (devMode) {
-      process.stdout.write(
-        `  ${chalk.bold.cyan(`${devMode.protocol}//localhost:${devMode.port}`)}`,
-      );
-      for (const ip of devMode.ips) {
-        process.stdout.write(
-          `${chalk.cyan(` > `)}${chalk.bold.cyan(`${devMode.protocol}//${ip}:${devMode.port}`)}`,
-        );
+      // SERVER_START event happened
+      if (startTimeMs > 0 && port > 0 && protocol) {
+        process.stdout.write(`  ${chalk.bold.cyan(`${protocol}//localhost:${port}`)}`);
+        for (const ip of ips) {
+          process.stdout.write(
+            `${chalk.cyan(` > `)}${chalk.bold.cyan(`${protocol}//${ip}:${port}`)}`,
+          );
+        }
+        process.stdout.write('\n' + chalk.dim(`  Server started in ${startTimeMs}ms.\n\n`));
+      } else {
+        // still waiting for SERVER_START
+        process.stdout.write(chalk.dim(`  Server starting…`) + '\n');
```

Comment:
While the server is starting

![Screen Shot 2020-06-12 at 12 42 35 PM](https://user-images.githubusercontent.com/1369770/84536039-39a8c100-acaa-11ea-85fa-d7ed5a17ce8c.png)

Reply:
Maybe we should add another line break?